util/cmd/cmd_test.go

```diff
@@ -3,6 +3,12 @@ package cmd
 import (
 	"reflect"
 	"testing"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
```

Comment:
```suggestion
	logtest "github.com/sirupsen/logrus/hooks/test"
```

we don't have another `test` here, but I feel like being specific is  useful here

Reply:
Okay, done in 5cff94056bd25bb9d961342de614ac8abaccd0aa